src/routes/[tripId]/rounds/[roundId]/+page.svelte

```diff
@@ -18,11 +20,15 @@
 	$: par = getPar(data.scorecard);
 	$: yardage = getYardage(data.scorecard);
 
+	$: tripPlayers = $players
+		.filter((player) => player.tripIds.includes(data.id))
+		.sort((a, b) => a.handicap - b.handicap);
```

Comment:
this logic is copied in two places now... probably could make the `tripPlayers` a derived store somehow that's aware of which trip you are in based on URL route.